docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: '3.0'
+services:
+  streetmerchant-shop-bot:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: streetmerchant-shop-bot
+    hostname: streetmerchant-shop-bot
```

```suggestion
  streetmerchant:
    build:
      context: .
      dockerfile: Dockerfile
    container_name: streetmerchant
```

I'd prefer we keep the name the same as the application. I also don't think it's necessary to have a `hostname`.